### ISSUE TITLE
package.json exports incorrect

### ISSUE THREAD

**Issue by wallw-teal** [jollyjerr/react-autosave#22]
`package.json` contains
```json
  "module": "./dist/react-autosave.es.js",
  "exports": {
    ".": {
      "import": "./dist/react-autosave.es.js",
    }
  },
```
But `dist/react-autosave.es.js` does not exist. Looks like it should be `dist/react-autosave.mjs` instead.

**Comment by TimMTech** [comment]
They need to close this off because this library is still clearly in beta.  please do not advertise broken libraries as it is a waste of time to people who need this in production.

**Comment by jollyjerr** [comment]
Fixed in 0.4.2
